SwiftCompilerSources/Sources/Optimizer/FunctionPasses/CopyToBorrowOptimization.swift

```diff
@@ -124,8 +124,11 @@ private struct Uses {
   // E.g. the none-case of a switch_enum of an Optional.
   private(set) var nonDestroyingLiverangeExits: Stack<Instruction>
 
-  var allLifetimeEndingInstructions: [Instruction] {
-    Array(destroys.lazy.map { $0 }) + Array(nonDestroyingLiverangeExits)
+  // FIXME: Use an InstructionSetVector
```

Comment:
An `InstructionWorklist` is basically an instruction set-vector. We will probably not add an `InstructionSetVector` in swift. So this comment doesn't make sense.

Reply:
It worked out here with inlining the worklist construction at the use-site. `InstructionSetVector` may still be valuable in some cases. To use a `InstructionWorklist` in this PR I had to originally copy it to a var and then iterate by popping the elements.